### ISSUE TITLE
`AggregateError: secretlint loading errors` on windows

### ISSUE THREAD

**Issue by kazupon** [secretlint/secretlint#205]
**What version of secretlint are you using?**

```sh
 npx envinfo --npmPackages secretlint

  npmPackages:
    secretlint: ^4.1.0 => 4.1.0
```

**What did you do? Please include the actual steps causing the issue.**

1. Upgrade to v4.1.0 from v3.3.0
2. Push commit, create PR, and Run the GitHub Actions

https://github.com/intlify/cli/pull/27/checks?check_run_id=3856182006

**What did you expect to happen?**

secretlint should work in windows environment with Github actions.
I checked the changelog when I upgraded to 4.0.0. but, I understood that I don't need to change my `.secretlintrc.json`. 
So, I have not made any changes.

**What actually happened? Please include the actual, raw output from secretlint.**

```
$ npx secretlint --debug "**/*"
2021-10-11T07:12:14.775Z secretlint input: [ '**/*' ]
2021-10-11T07:12:14.776Z secretlint flags: {
  debug: true,
  init: false,
  format: 'stylish',
  secretlintignore: '.secretlintignore',
  color: true,
  terminalLink: true,
  maskSecrets: false,
  profile: false,
  cwd: 'D:\\a\\cli\\cli'
}
2021-10-11T07:12:14.777Z secretlint searchFiles ignore baseDir: , normalizeIgnoreFilePath: D:\a\cli\cli\.secretlintignore
2021-10-11T07:12:14.778Z secretlint search patterns: [ '**/*' ]
2021-10-11T07:12:14.778Z secretlint search ignore patterns: [ '**/.git/**', '**/node_modules/**', '**/.secretlintrc/**', '**/.secretlintrc.{json,yaml,yml,js}/**', '**/.secretlintignore*/**' ]
AggregateError: secretlint loading errors

Only file and data URLs are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'd:'
    at loadPackagesFromRawConfig (D:\a\cli\cli\node_modules\@secretlint\config-loader\lib\index.js:87:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async loadConfig (D:\a\cli\cli\node_modules\@secretlint\config-loader\lib\index.js:120:20)
    at async D:\a\cli\cli\node_modules\@secretlint\node\lib\index.js:116:34
    at async createEngine (D:\a\cli\cli\node_modules\@secretlint\node\lib\index.js:109:26)
    at async runSecretLint (D:\a\cli\cli\node_modules\secretlint\lib\index.js:18:20) {
  errors: [
    Error [ERR_UNSUPPORTED_ESM_URL_SCHEME]: Only file and data URLs are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'd:'
        at new NodeError (node:internal/errors:371:5)
        at Loader.defaultResolve [as _resolve] (node:internal/modules/esm/resolve:1016:11)
        at Loader.resolve (node:internal/modules/esm/loader:89:40)
        at Loader.getModuleJob (node:internal/modules/esm/loader:242:28)
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
        at Loader.import (node:internal/modules/esm/loader:177:28)
        at importModuleDynamically (node:internal/modules/cjs/loader:1041:29)
        at importModuleDynamicallyWrapper (node:internal/vm/module:437:21)
        at importModuleDynamically (node:vm:381:43)
        at exports.importModuleDynamicallyCallback (node:internal/process/esm_loader:30:14)
        at eval (eval at <anonymous> (D:\a\cli\cli\node_modules\@secretlint\config-loader\lib\index.js:28:24), <anonymous>:3:1) {
      code: 'ERR_UNSUPPORTED_ESM_URL_SCHEME'
    }
  ]
}
error Command failed with exit code 1.
ERROR: "lint:secret" exited with 1.
error Command failed with exit code 1.
```

**Comment by azu** [comment]
> Only file and data URLs are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'd:'

Probably, it realte to https://github.com/secretlint/secretlint/pull/187
It looks like that `import("D://.path")`  throw an Error.

We need to convert it to `file://` protocol, maybe.

**Comment by azu** [comment]
maybe https://nodejs.org/api/url.html#url_url_pathtofileurl_path

```diff
- await _importDynamic(moduleResolver.resolveRulePackageName(configDescriptorRule.id))
+ await _importDynamic(url.pathToFileURL(moduleResolver.resolveRulePackageName(configDescriptorRule.id)))
```
